packages/web-react/src/components/Tag/__tests__/useTagStyleProps.test.ts

```diff
@@ -10,7 +10,7 @@ describe('useTagStyleProps', () => {
     expect(result.current.classProps).toBe('Tag');
   });
 
-  it.each([['neutral'], ['default'], ['success'], ['informative'], ['warning'], ['danger']])(
+  it.each([['neutral'], ['success'], ['informative'], ['warning'], ['danger']])(
```

Comment:
Can we use one of the dictionary prop tests?

Reply:
This is the hook test. AFAIK we currently don't have global tests for these. (We have them for the components tests)

Reply:
Ok, my fault.